vi-start-to-finish.py

```diff
@@ -0,0 +1,223 @@
+"""Use dask-jobqueue to correct illumination, montage, and extract features."""
+import os
+import itertools
+import glob
+import sys
+import re
+import time
+import click
+import toolz as tz
+import numpy as np
+import pandas as pd
+from dask_jobqueue import SLURMCluster as Cluster
+from dask import delayed
+from dask.distributed import Client, as_completed
+from distributed.scheduler import KilledWorker
+from tqdm import tqdm
+from skimage import io
+from microscopium import preprocess as pre
+from microscopium._util import generate_spiral
+from microscopium import features as feat
+
+
+ROOT = '/scratch/su62/petermac/data'
+os.chdir(ROOT)
+OUT = os.path.join(ROOT, 'out')
+os.makedirs(OUT, exist_ok=True)
+
+
+def ftime(seconds):
+    string = ''
+    units = ['d', 'h', 'm', 's']
+    values = [3600 * 24, 3600, 60, 1]
+    for unit, value in zip(units, values):
+        amount = seconds // value
+        if amount > 0:
+            string += f'{amount}{unit}'
+        seconds = seconds % value
+    return string
+
+# config in /home/jnun0003/.config/dask/jobqueue.yaml
+cluster = Cluster(walltime='06:00:00')
+cluster.adapt(minimum=2, maximum=16, target_duration='1d')
+
+############### illumination fields #####################
+t0 = time.time()
+
+exp = (r'(?P<dir>.*)/(?P<plate>mfgtmp_\d*)_'
```

Comment:
This is also specific to this particular dataset, can you group it with the other user inputs?

Reply:
This definition was left over from an earlier version of the script, removed.